### PR TITLE
[MM-28500] Make clear that slash command don't automatically split into posts

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Instead educate about the existence of `extra_responses`.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28500
Fixes https://github.com/mattermost/mattermost-server/issues/15384

### CODE REVIEW

File: source/developer/slash-commands.rst [hunk 1]
```diff
@@ -150,7 +150,7 @@ Tips and Best Practices
 
 1. Slash commands are designed to easily allow you to post messages. For other actions such as channel creation, you must also use the `Mattermost APIs <https://api.mattermost.com>`__.
 
-2. If the text is longer than the allowable character limit per post, the message is split into multiple consecutive posts, each within the character limit. Servers running Mattermost Server v5.0 or later `can support posts up to 16383 characters <https://docs.mattermost.com/administration/important-upgrade-notes.html>`__.
+2. Posts size is limited to 16393 characters for servers running `Mattermost Server v5.0 or later <https://docs.mattermost.com/administration/important-upgrade-notes.html>`__. Use the `extra_responses` field to reply with more the one post to a triggered slash command.
```
Comment (justinegeffen) [inline source/developer/slash-commands.rst:153]:
```suggestion
2. Posts size is limited to 16393 characters for servers running `Mattermost Server v5.0 or later <https://docs.mattermost.com/administration/important-upgrade-notes.html>`__. Use the `extra_responses` field to reply to a triggered slash command with more than one post.
```

**Review by justinegeffen — APPROVED** [review]
One suggestion otherwise LGTM!

**Review by justinegeffen — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Comment by larkox** [comment]
Maybe a link to the documentation where extra responses is explained?

**Comment by hanzei** [comment]
> Maybe a link to the documentation where extra responses is explained?

Well, that would be https://developers.mattermost.com/integrate/slash-commands/#parameters. I'm leaning forward not linking to a different site. @larkox What do you think?

@justinegeffen That is another example of an unclear audience for https://developers.mattermost.com vs. https://docs.mattermost.com

**Comment by larkox** [comment]
@hanzei I personally like things to be linked, even if between different sites, but I understand your point. I do not have a strong opinion, so I will leave it to you.

**Comment by justinegeffen** [comment]
Thanks for picking that up, @hanzei, very good point. I'm 3/5 for linking to https://developers.mattermost.com/integrate/slash-commands/#parameters just for reference. But agree that the audience is not well-defined.

**Pull request merged**